spec/unit/resource/api1600/synergy/volume_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1200::Synergy::Volume do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1200::C7000::Volume' do
+    expect(described_class).to be < OneviewSDK::API1200::C7000::Volume
```

Comment:
I think you should inherit from API1600::C7000::Volume. If so, please update description also accordingly.

Reply:
done